model/host/hostutil.go

```diff
@@ -949,8 +949,20 @@ func (h *Host) SetupSpawnHostCommands(settings *evergreen.Settings) (string, err
 
 	script := setupBinDirCmds
 	if h.ProvisionOptions.TaskId != "" {
-		fetchCmd := fmt.Sprintf("%s -c %s fetch -t %s --source --artifacts --dir='%s'", binaryPath, confPath, h.ProvisionOptions.TaskId, h.Distro.WorkDir)
-		script += " && " + fetchCmd
+		fetchCmd := []string{
+			binaryPath,
+			"-c", confPath,
+			"fetch",
+			"-t", h.ProvisionOptions.TaskId,
+			"--source",
+			"--artifacts",
+			"--dir", h.Distro.WorkDir,
```

Comment:
This changes the syntax from `--dir='%s'` to `--dir %s`.  Is that intentional/does that work?

Reply:
This is the same - shell syntax is terrible so I used single quotation marks to force it to interpret the string literally (a bash-ism). Using Jasper causes it to go directly to fork/exec instead of going through the bash parser first.